documents/testing/sections/Issues.tex

```diff
@@ -0,0 +1,60 @@
+\newcommand{\bug}[3]{\begin{minipage}{\textwidth}\textbf{Bug:} #1 \\ \textbf{Identified cause:} #2 \\ \textbf{Implemented fix:} #3 \end{minipage}\vskip.2\baselineskip}
+\newcommand{\knownissue}[3]{\begin{minipage}{\textwidth}\textbf{Prerequisite:} #1 \par\vspace{4pt} \textbf{Actual behavior:} #2 \par\vspace{4pt} \textbf{Desired behavior:} #3 \end{minipage}\vskip.2\baselineskip}
+
+\chapter{Issues}
```

Comment:
```suggestion
\chapter{Resolved issues}
```
To have a clear distinction between this chapter and the following `Known issues`. Alternative: `Fixed issues`.